src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -560,7 +560,7 @@ def _lookup_kformat(key_type: bytes):
     raise UnsupportedAlgorithm(f"Unsupported key type: {key_type!r}")
 
 
-_SSH_PRIVATE_KEY_TYPES = typing.Union[
+SSHPrivateKeyTypes = typing.Union[
```

Comment:
@reaperhulk suggested the name `SSHPublicKeyTypes`, I wasn't sure whether I should retain the underscore or not.

Reply:
What's the convention? You previously told us that UPPER_SNAKE_CASE wasn't the normal pattern for Python projects -- we should follow whatever the normal pattern is 😄

Reply:
If the type aliases can be useful for downstream projects (I believe most of these are), I think they should be unprefixed and declared a public API.

So the only piece missing from that is documentation?

With exceptions I think being `_IPAddressTypes` (since these types are defined outside of cryptography) and `_AEADTypes` (which is only defined with `if TYPE_CHECKING` and thus would be difficult to explain how to use correctly). The latter could be turned into runtime imports as well.

Reply:
Okay, let's add documentation then!